### DISCUSSION TITLE
Nanoid in JSON

### DISCUSSION THREAD

**Discussion by devchristiansimon** [neuefische/questions#19] [Web] [answered]
Ich habe eine separate JSON Datei mit Hard Coded content und möchte da Nanoid einfügen. Allerdings funktioniert der Import nicht bei JSON Dateien, daher die Frage: Wie kann ich nanoids innerhalb von JSON Dateien vergeben?

Danke im voraus <3

**Comment by chris-yooo** [comment, marked as answer]
ich glaub das geht nicht du kannst aber die json erstellen und nano id verwenden um in die json eine ID reinzuschreiben.

aber innerhalb der json nanoid zu verwenden haut nich hin i think

**Reply by stefankorder** [reply]
Eine nanoid lässt sich nur außerhalb des json hinzufügen 👍🏻😃
